components/schemas/stacks/builds/taskActions/DeployStackBuildAction.yml

```diff
@@ -1,22 +1,22 @@
 title: DeployStackBuild
-type: object 
-required: 
-  - action 
-  - contents 
-properties: 
-  action: 
-    type: string 
-    description: The action the job takes. 
-    enum: 
+type: object
+required:
+  - action
+  - contents
+properties:
+  action:
+    type: string
+    description: The action the job takes.
+    enum:
       - deploy
-  contents: 
-    type: object 
-    description: Additional information needed for the job. 
-    required: 
-      - environment_id 
-    properties: 
-      environment_id: 
-        type: string 
-        description: The ID for the environment this stack build will be deployed to. 
-      update: 
-        $ref: "../StackBuildContainerUpdate.yml"
\ No newline at end of file
+  contents:
+    type: object
+    description: Additional information needed for the job.
+    required:
+      - environment_id
+    properties:
+      environment_id:
+        type: string
+        description: The ID for the environment this stack build will be deployed to.
+      update:
```

Comment:
needs to be nullable too.
![image](https://github.com/cycleplatform/api-spec/assets/5110855/11d14706-93df-40d7-83cc-88d193921f24)